Util/BuildTools/Setup.sh

```diff
@@ -166,8 +181,12 @@ else
 
   log "Retrieving rpclib."
 
+  start_download_time=$(date +%s)
+
   git clone -b ${RPCLIB_PATCH} https://github.com/carla-simulator/rpclib.git ${RPCLIB_BASENAME}-source
 
+  end_download_time=$(date +%s)
+  echo "Elapsed Time downloading rpclib: $(($end-$start)) seconds"
```

Comment:
Seems this line should be like this:

```bash
echo "Elapsed Time downloading rpclib: $(($end_download_time-$start_download_time)) seconds"
```

As end and start variables have different names here.